src/client/headers/shared/types.hpp

```diff
@@ -1003,7 +1054,7 @@ namespace intercept {
                 *reinterpret_cast<uintptr_t*>(this) = type_def;
                 *reinterpret_cast<uintptr_t*>(static_cast<I_debug_value*>(this)) = data_type_def;
             }
-            size_t hash() const { return __internal::pairhash(type_def, location); }
+            size_t hash() const { return this->get_hash(); }
```

Comment:
```suggestion
           [[deprecated("Use get_hash instead")]]  size_t hash() const { return this->get_hash(); }
```


Actually now after making all these suggestions.
You can just plainly remove all these, and add

`[[deprecated("Use get_hash instead")]]  size_t hash() const { return this->get_hash(); }`
directly into game_data instead

Reply:
Yeah idk what I was thinking 😅